x-pack/plugins/apm/server/feature.ts

```diff
@@ -67,60 +66,6 @@ export const APM_FEATURE = {
       ui: ['show', 'alerting:show'],
```

Comment:
I can't comment directly on the lines affected, but if you're removing the subfeature privilege, you will have to update the `privileges.all.alerting` and `privileges.read.alerting` to include `alert` as well as `rule`, so

```
privileges: {
  all: {
    alerting: {
      rule: {
        all: Object.values(AlertType),
      },
       alert: {
        all: Object.values(AlertType),
      }
    }
  },
  read: {
    alerting: {
      rule: {
        read: Object.values(AlertType),
      },
       alert: {
        read: Object.values(AlertType),
      }
    }
  }
}
```